wf/ja-jp/content/topics/workflows/connector-builder/authentication-custom.htm

```diff
@@ -347,35 +323,15 @@ <h1>カスタム認証の構築</h1>
-                                                                    <ul>
-                                                                        <li>
-                                                                            <p><span class="code">静的な値</span>であれば、［Value（値）］フィールドに値を入力します。</p>
-                                                                        </li>
-                                                                        <li>
-                                                                            <p><span class="code">ヘルパーフローからの値</span>であれば、<span class="wintitle">［Choose Flow（フローの選択）］</span>ダイアログでフローを選択します。</p>
-                                                                        </li>
-                                                                    </ul>
-                                                                    <p><span class="uicontrol">Type（タイプ）</span>：フィールドには2つのタイプがあります。</p>
-                                                                    <ul>
-                                                                        <li>
-                                                                            <p><span class="uicontrol">Text（テキスト）</span>：ユーザーにはプレーンテキストフィールドが表示されます。これは、加工なしのオープンテキストフィールドです。</p>
-                                                                        </li>
-                                                                        <li>
-                                                                            <p><span class="uicontrol">Password（パスワード）</span>：加工されたテキストフィールド。ユーザーには、このフィールドの入力内容は表示されません。</p>
-                                                                        </li>
-                                                                    </ul>
-                                                                    <p><span class="uicontrol">［Visible（表示）］</span>トグルは、コネクターの認証ダイアログに構成値を表示するかどうかを決定します。</p>
-                                                                </td>
+        コネクターに対してユーザーを認証する方法を定義するために、その他の設定値が必要になる場合があります。<span class="uicontrol">［Add Config Value（構成値を追加）］</span>をクリックします。
+        <p><span class="uicontrol">Label（ラベル）</span>：コネクターのセットアップ時に<span class="wintitle">［New Connection（新規接続）］</span>ダイアログに表示される構成値の名前。</p><p><span class="uicontrol">Definition（定義）</span>：このフィールドに静的な値が入力されるか、ヘルパーフローからの値が入力されるかを示します。</p><ul><li><p><span class="userinput">静的な値</span>であれば、<span class="uicontrol">［Value（値）］</span>フィールドに値を入力します。</p></li><li><p><span class="userinput">ヘルパーフローからの値</span>であれば、<span class="wintitle">［Choose Flow（フローの選択）］</span>ダイアログでフローを選択します。</p></li></ul><p><span class="uicontrol">Type（タイプ）</span>：2つのフィールドタイプから選択できます。</p><ul><li><p><span class="userinput">Text（テキスト）</span>：ユーザーにプレーンテキストを表示します。これは、加工なしのオープンテキストフィールドです。</p></li><li><p><span class="userinput">パスワード</span>：加工されたテキストフィールドを表示します。このフィールドに入力されたテキストの内容はユーザーにはわかりません。</p></li></ul><p><span class="uicontrol">［Visible（可視性）］</span>トグルは、コネクターの認証ダイアログに構成値を表示するかどうかを決定します。</p></td>
```

Comment:
Some of our UI control tags have brackets, and some don't, here and elsewhere. It'd be nice to get rid of the hard-coded brackets everywhere (not today).